packages/chain-adapters/src/utxo/UtxoBaseAdapter.ts

```diff
@@ -130,10 +130,6 @@ export abstract class UtxoBaseAdapter<T extends UtxoChainId> implements IChainAd
     return this.coinName
   }
 
-  buildBIP44Params(params: Partial<BIP44Params>): BIP44Params {
```

Comment:
remove this, as it doesn't have any validation, and having both `build` and `get` is a bit derp

Reply:
this is only used in 3 places in web so it's trivial to swap out

Reply:
ok 22 but that's basically 3